### PR TITLE
Upgrade Logstash

### PR DESCRIPTION
**Pull request by djsd123**
Changing image tag to match elasticsearch version

### CODE REVIEW

File: charts/logstash/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ replicaCount: 1
 
 image:
   repository: docker.elastic.co/logstash/logstash
-  tag: 5.6.10
+  tag: 6.3.1
```
Comment (xoen) [inline charts/logstash/values.yaml]: Any breaking change in this?

File: charts/logstash/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] - 2018-07-09
+## Upgrade Logstash
+- Modifying Image tag to match elasticsearch version
```
Comment (xoen) [inline charts/logstash/CHANGELOG.md]: I guess it's overkill as we can always go back, check Git history, etc...but, I find useful to have version changes in the Changelog as well, e.g.

> Bumping image tag to `6.3.1` (from `5.6.10`) to match elasticsearch version

File: charts/logstash/values.yaml (hunk 1 above)
Comment (djsd123) [reply charts/logstash/values.yaml]: Breaks without it

File: charts/logstash/CHANGELOG.md (hunk 2 above)
Comment (djsd123) [reply charts/logstash/CHANGELOG.md]: 👍

**Pull request merged**